spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/filtering.adoc

```diff
@@ -27,3 +27,62 @@ public void listen(Thing thing) {
 }
 ----
 
+### Ignore empty batch when you use `batch` mode.
+Starting with version `3.3.0`, we support ignoring empty batch that result from filtering by `RecordFilterStrategy`.
+When implementing `RecordFilterStrategy`, you can configure it through `ignoreEmptyBatch()`. default is `false`, that means `KafkaListener` will be invoked even if all `ConsumerRecord` are filtered out.
+
+If `true` is returned, the `KafkaListener` will not be invoked when all `ConsumerRecord` s are filtered out. However, commit to broker, will still be executed.
+If `false` is returned, the `KafkaListener` will be invoked when all `ConsumerRecord` s are filtered out.
+
+Let's look some examples.
```

Comment:
`Let's` is also not a technical documentation word 😄

Reply:
I see!! 😄😄